process/incident_management.md

```diff
@@ -0,0 +1,263 @@
+# Incident Management Process
+
+## Introduction
+The goal of this page is to document the incident management process.
+
+It aims at formalizing the roles of the different actors while working on a **major** incident, as well as describing the high level course of actions and mandatory steps that need to be taken to minimize MTTR (=Mean Time To Recover/Repair, so ensuring quick recovery of our services) and proper customer communication and satisfaction throughout the process.
+
+There can be multiple levels of severity for an incident depending on the impact it has. The process below would be used when the incident is considered to be a major incident. When the incident is a minor one then a lightweight ad-hoc process can be used. The goal being to make sure not to make the process too heavy for small incidents.
+
+A major outage can be defined when our platform is running out of error budget or when we start failing our SLOs.
+
+Examples of major incidents:
+* Full outage or heavy degradation of one or multiple services impacting one or several customers
+* Duration of the incident is greater than 4 hours
+* Multiple layers and SRE teams are impacted (platform, application, …)
+
+Note that when a team has a path for fast recovery then they shouldn't break it to fit that process if it's not required.
+
+## Quick reference guide
+In case of a customer–impacting incident (persisting or intermittent degradation of customer–exposed functionality):
+
+1. The Incident First Responder is the first SRE being alerted of the issue and starting the investigation
```

Comment:
This is the first time an 'Incident First Responder' role is mentioned. Would it be helpful to add the personas/roles section above this one?